.travis.yml

```diff
@@ -6,20 +6,7 @@ python:
 cache: pip
 
 install:
-  - sudo apt-get update
-  # https://conda.io/docs/user-guide/tasks/use-conda-with-travis-ci.html
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
-  - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
-  # Useful for debugging any issues with conda
-  - conda info -a
-  - conda env create -f environment.yml
-  - source activate earth-analytics-python
```

Comment:
The thing that i'm worried about here is that our envt is specific to this library. Many of the dependencies - like GDAL won't install if people start using pip. conda-forge is the main install repo. i just don't understand how this will work without an explicit envt!

Reply:
I think I am following, and I totally hear you that conda-forge would be the go-to place from which to install earthpy's dependencies. Having unit tests will help us to figure out whether we get different answers if we install using pip vs. conda/conda-forge.

As far as multiple environments - I think the goal here is to capture earthpy's dependencies in this repo, and capture your earth analytics class dependencies elsewhere. That way, we check two boxes:

1. Students can still install your conda environment, then install this package, and have most or all of earthpy's dependencies installed already. 

2. Users of this package do not need to install the full earth-analytics-python conda environment (e.g., if you only wanted to use earthpy, and not install all of the other packages that get used in the course).

Reply:
The way I understand Max's comment: have earthpy (this package) specify its dependencies in `setup.py` plus instructions for manual steps (often this is left to the reader "Make sure you have GDAL>=vX.Y, how you get it is up to you") for things that can't be pip installed (e.g. GDAL which is needed by rasterio).

Then we have the earth-analytics-python which is a ready made environment that has an opinion on how you should install stuff (conda from conda-forge). In practice not much would have to change in `earthlab/earth-analytics-python-env` as it `pip` installs earthpy which if all the dependencies are already satisfied will just work.

Overall I think Max's plan sounds good.